packages/mongo/scripts/seed.ts

```diff
@@ -0,0 +1,159 @@
+/**
```

Comment:
@agoldis just a seed script, no impact on prod.
added tsconfig-paths/register in devDependencies